hypha/apply/funds/models/submissions.py

```diff
@@ -536,6 +536,10 @@ def get_from_parent(self, attribute):
             # We are a lab submission
             return getattr(self.page.specific, attribute)
 
+    @property
+    def is_determination_form_attached(self):
+        return self.get_from_parent('determination_forms').count() > 0
```

Comment:
Working smoothly, great work. Noticed an issue with Lab. A lab works differently from Funds. A Fund has Rounds. So, we are able to use new determination streamfield forms for new Rounds and current active/old rounds can use old forms because they do not have the determination streamfield forms set. And there is no way to edit round as well. So, we are safe here. But Labs do not have Rounds. So, In admin when a staff user attaches new determination streamfield form to a lab, old submissions for a lab also start showing new forms instead of old. So, we need a come up with an approach for labs.